dist/parcoords.standalone.js

```diff
@@ -8577,6 +8577,10 @@
         ctx.brushed.scale(devicePixelRatio, devicePixelRatio);
         ctx.highlight.lineWidth = 3;
         ctx.highlight.scale(devicePixelRatio, devicePixelRatio);
+        ctx.marked.lineWidth = 3;
+        ctx.marked.shadowColor = "#ffffff";
+        ctx.marked.shadowBlur = 10;
+        ctx.marked.scale(devicePixelRatio, devicePixelRatio);
```

Comment:
Hmm... Would you please make `shadowColor`, `shadowBlur` and marker's `lineWidth` in configurations?

I'll make highlight's `lineWidth` read from configuration later.

Reply:
I've got the updated code almost finished. I'm not sure what you mean by making these in configurations, however. Could you sketch up the `highlight.linewidth` example in a reply so I can replicate that with marked?

Reply:
I mean end uses can customize `lineWidth`, `shadowColor` and `shadowBlur`.

Steps are:
1. Add definitions to [DefaultConfig](https://github.com/BigFatDog/parcoords-es/blob/develop/src/state/defaultConfig.js):
```
const DefaultConfig = {
...
  color: '#069',
  markerLineWidth: 3, 
  markerShadowColor: '#ffffff',
  markerShadowBlur: 10
...
}
```
Naming depends on your design: 
* If you plan to let marker feature share the global `lineWidth` or `shadowColor` with other features (normal line width, highlighted line width, etc.), please keep your original design, I'll update other features accordingly.
* If you'd like to reserve these config for marker feature only, `markerShadowBlur` should be a better choice.

2. Refer config. Here is an example: [usage of `dimensionTitleRotation`](https://github.com/BigFatDog/parcoords-es/blob/f41c1b1774b45718ff81890eb4fed8f010a365a8/src/api/updateAxes.js#L51)

3. End users customize:
```
pc()
.markerLineWidth(6)
.markerShadowColor('#ffffff')
.markerShadowBlur(10)
```

Reply:
Ok great! I think I've got it all setup. I'm going to try to build and test the code and hopefully I'll have the pull request ready by the end of the day.